gpu_plots.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
```

Comment:
I might switch to matplotlib, my use of gnuplot is laziness.

Reply:
I have a love/hate relationship with matplotlib but it has worked well for me.